mautic/mautic
# Fix contact name  campaign fail log on send marketing email

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
If you send multiple emails by campaigns, and use marketing  message. The fail log in campaign return wrong contact name. 


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Create campaign with two send email action as marketing message
2.  Add two contacts
3. Send it and see  fail notification for both contact in notification area. Both has same name as contacnt

![image](https://user-images.githubusercontent.com/462477/45971821-53b7f100-c03a-11e8-8714-a4604084ffa6.png)


#### Steps to test this PR:
1.  Repeat all steps and see notifciation return name for each contact as we excepted
